### DISCUSSION TITLE
Make translations files available on Crowdin

### DISCUSSION THREAD

**Discussion by srt54558** [d4rken-org/sdmaid-se#530] [General]
Hello, 
I'd like to help with the German translation but there are no files to display on crowdin. When can we start contributing on the translation?

**Comment by d4rken** [comment]
I hope to have the translation files ready next week. I'm preparing stuff still, and I will likely have to remove SD Maid 1 translations to make space.

**Comment by d4rken** [comment]
Translation files are available: https://crowdin.com/project/sdmaid-se
